### ISSUE TITLE
setTimeout, etc?

### ISSUE THREAD

**Issue by tlrobinson** [audreyt/node-webworker-threads#50]
Workers don't have access to setTimeout, setInterval, etc.

**Comment by audreyt** [comment]
Indeed, the [HTML timer spec](http://www.w3.org/TR/2011/WD-html5-20110525/timers.html) is not native to v8 and will need to be re-implemented in C++ level.

**Comment by andywer** [comment]
Would need this functionality, too. Is there a work-around for now?

**Comment by audreyt** [comment]
Yes, if it's OK to ignore the milliseconds...

```js
function setTimeout (cb, ms) { return thread.nextTick(cb); }
```

**Comment by mitar** [comment]
This is to me the largest issue with web workers on node.js. Also, one thing to keep in mind is that `terminate` has to terminate also all current timers, but only for that particular web workers. This is an issue for example currently, if one wants to use `vm` to do some type of web workers, is that `vm` does not have `terminate`.

**Comment by felixbanguera** [comment]
Hi @audreyt ,
when you mention the work around for setTimeout is: 
`function setTimeout (cb, ms) { return thread.nextTick(cb); }`

Can you writhe a functional example of this? not sure what you mean with `cb`.

What I'm trying to do on tests purposes is to emit a thread event with a delay.

thks
